src/Chainweb/Version.hs

```diff
@@ -888,9 +889,16 @@ pact4coin3Upgrade aoa v h = case aoa of
     go f Mainnet01 = f 1_722_500 -- 2021-06-19T03:34:05
     go f Testnet04 = f 1_261_000 -- 2021-06-17T15:54:14
     go f Development = f 80
-    go f (FastTimedCPM g) | g == petersonChainGraph = f 20
+    go f (FastTimedCPM g) | g == petersonChainGraph = f 15
```

Comment:
I think this change is probably unnecessary given that the pact 4.1 test didn't change.